### PR TITLE
Working on issue #75

### PR DESCRIPTION
**Pull request by alegambo**

### CODE REVIEW

File: cruds_adminlte/crud.py [hunk 1]
```diff
@@ -165,7 +165,7 @@ def dispatch(self, request, *args, **kwargs):
         for perm in self.perms:
             if not self.validate_user_perms(request.user, perm,
                                             self.view_type):
-                return HttpResponseForbidden()
+                return HttpResponseForbidden(render(request, 'HttpResponse/403.html'))
```
Comment (oscarmlage) [inline cruds_adminlte/crud.py]: Make sure this is pep8 compliant please.

File: cruds_adminlte/templates/HttpResponse/403.html [hunk 2]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```
Comment (oscarmlage) [inline cruds_adminlte/templates/HttpResponse/403.html]: I think this new template should follow the template patterns, please take a look to `templates/adminlte/example.html` to see how to extend from `templates/adminlte/base.html` and just change the titles and the body.

File: cruds_adminlte/crud.py (hunk 1 above)
Comment (luisza) [reply cruds_adminlte/crud.py]: mmm maybe something like 

`return HttpResponseForbidden(render_to_string('403.html'), request=request)`

File: cruds_adminlte/templates/HttpResponse/403.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    </style>
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+        <div id="error-page" class="col-md-12">
+            <h2>Error 403</h2>
+            <h2>You are not authorized to view this page</h2>
+            <br>
+            <a class="btn btn-primary" href="/">Return to the main page</a>
```
Comment (luisza) [inline cruds_adminlte/templates/HttpResponse/403.html]: It's required to include locate implemententation,  try to use 
{% load i18n %} in the top of file and something like 
{% trans 'Return to the main page' %}

File: cruds_adminlte/templates/HttpResponse/403.html [hunk 4]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
```
Comment (luisza) [inline cruds_adminlte/templates/HttpResponse/403.html]: Try to extend from base.html, find some html on templates/cruds/ and look how to extends from base file.

**Pull request merged**